apps/gitness/src/components-v2/file-content-viewer.tsx

```diff
@@ -42,6 +43,9 @@ export default function FileContentViewer({ repoContent }: FileContentViewerProp
   const [view, setView] = useState<ViewTypeValue>(getDefaultView(language))
   const [isDeleteFileDialogOpen, setIsDeleteFileDialogOpen] = useState(false)
   const { selectedBranchTag } = useRepoBranchesStore()
+  const { theme } = useThemeStore()
```

Comment:
We should probably alter `useThemeStore` to _always_ have a theme instead of potentially being `undefined`. Not a problem for this PR